app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,7 @@
+class FoodsController < ApplicationController
+  def index
+    params[:q].present?
+      @search_results = FoodFacade.search_by_ingredient(params[:q])
+    @search_term = params[:q]
+  end
```

Comment:
In addition to the search results, you'll also want to find a way to print out the total hits that came back from the endpoint.